sentry-config.cmake.in

```diff
@@ -1,6 +1,10 @@
 set(SENTRY_BACKEND @SENTRY_BACKEND@)
 set(SENTRY_TRANSPORT @SENTRY_TRANSPORT@)
 
+if(SENTRY_BACKEND STREQUAL "crashpad")
+	find_package(crashpad REQUIRED)
```

Comment:
this probably also needs to be guarded by `SENTRY_CRASHPAD_SYSTEM`, no?

Reply:
Well, that's one thing I'm not so sure about.
When defining `CRASHPAD_ENABLE_INSTALL` (and NOT `CRASHPAD_ENABLE_INSTALL_DEV`), `crashpad` will install all libraries and the `crashpad-config.cmake` script.

In `sentry-native`, I then link to `crashpad::client`, which needs to be present by a consumer.
So in `sentry-native-config.cmake`, we need to also do `find_package(crashpad REQUIRED)`.

So after this pr, an installation of `sentry-native` with a vendored crashpad, will install the sentry libraries, sentry headers, sentry-native-config.cmake and crashpad-config.cmake.

Reply:
I looked at it again and it is fixed.
The targets of crashpad are now exported to `sentry_crashpad-targets.cmake` and included appropriately.